File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        rev_num=0
        org_num=x
        while(x>0):
            lastdigit = x%10
            rev_num = (rev_num*10)+ lastdigit
            x//=10
        if(rev_num==org_num):
            return True
        else:
            return False

File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        int dup = x;
        long int rev=0;
        while(x>0){
            int last=x%10;
            rev=(rev*10)+last;
            x/=10;
        }      
        if(dup==rev) return true;
    return false;
    }
};